### DISCUSSION TITLE
Found Cloudflare Cookie!

### DISCUSSION THREAD

**Discussion by slamd14** [acheong08/ChatGPT#406] [Q&A] [answered]
# After I run the __main__, the chrome comes out succssfully. But the notice below comes out on my console:

```
Spawning browser...
Browser spawned.
Found Cloudflare Cookie!
Spawning browser...
Browser spawned.
Found Cloudflare Cookie!
Spawning browser...
...
```

**Comment by acheong08** [comment, marked as answer]
There are sometimes issues due IPv6, browser versions, etc. There is no proper fix right now because I can't replicate the error. Use https://github.com/acheong08/ChatGPT-lite in the meantime which avoids the browser.

**Reply by slamd14** [reply]
Thanks bro!!! ChatGPT-lite is useful for me. 
It came out a error: "binascii.a2b_base64(s) binascii.error incorrect padding" when I just asked the bot "hi"🤣, I know the error is caused by the base64_decode, so I tried to replace the code in ChatGPT.py ```parsed = json.loads(base64.b64decode(token.split('.')[1]).decode())``` by  ``` parsed = json.loads(base64.b64decode(token.split('.')[1] + "===").decode())```. And it is effective.

**Reply by acheong08** [reply]
Thanks. I will fix this.

**Reply by kaifeijidezxb** [reply]
Hello, would like to know the version of the browser you chose, I have this problem, too I don't know which version is the correct one,
